httpgrpc/server.go

```diff
@@ -101,7 +178,11 @@ func HandleMethod(svr interface{}, serviceName string, desc *grpc.MethodDesc, un
 		if err != nil {
 			st, _ := status.FromError(err)
 			if st.Code() == codes.OK {
-				st = status.New(codes.Internal, err.Error())
```

Comment:
This old formulation was losing any details. So now we preserve them and basically just convert to a new status proto where only the code is changed.